WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutPreviewFragment.kt

```diff
@@ -39,47 +39,62 @@ abstract class LayoutPreviewFragment : FullscreenBottomSheetDialogFragment() {
     private lateinit var viewModel: LayoutPickerViewModel
     private lateinit var previewModeSelectorPopup: PreviewModeSelectorPopup
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
-            inflater.inflate(R.layout.layout_picker_preview_fragment, container)
+    private var _binding: LayoutPickerPreviewFragmentBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = LayoutPickerPreviewFragmentBinding.inflate(inflater, container, false)
+        return binding.root
+    }
 
     @SuppressLint("SetJavaScriptEnabled")
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        webView.settings.javaScriptEnabled = true
-        webView.settings.useWideViewPort = true
-        webView.setInitialScale(INITIAL_SCALE)
+        binding.webView.settings.javaScriptEnabled = true
+        binding.webView.settings.useWideViewPort = true
+        binding.webView.setInitialScale(INITIAL_SCALE)
+        binding.chooseButton.setText(getChooseButtonText())
+        setViewModel(getViewModel())
```

Comment:
Minor (🔍): Consider renaming that to `initViewModel()` and moving the `getViewModel()` assignment within that function itself, since there is no real need to pass that as a parameter and further complicate the function's signature:
```
this.viewModel = getViewModel()
```

Reply:
Yep, I agree, this is minor. But if I am in there anyway. :)

Reply:
Updated in 2e428c2